usr.sbin/makefs/ffs.c

```diff
@@ -882,6 +882,7 @@ ffs_write_file(union dinode *din, uint32_t ino, void *buf, fsinfo_t *fsopts)
 	struct inode	in;
 	struct buf *	bp;
 	ffs_opt_t	*ffs_opts = fsopts->fs_specific;
+	struct vnode vp = { fsopts, NULL };
```

Comment:
Ok, it seems that `v_data` is a necessary field of `vnode` as it is used in msdos support.